pkg/cap/anchor/cluster/server.go

```diff
@@ -0,0 +1,135 @@
+package cluster
+
+import (
+	"context"
+	"time"
+
+	"capnproto.org/go/capnp/v3/server"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/wetware/casm/pkg/cluster"
+	"github.com/wetware/casm/pkg/cluster/routing"
+	api "github.com/wetware/ww/internal/api/cluster"
+)
+
+var defaultPolicy = server.Policy{
+	// HACK:  raise MaxConcurrentCalls to mitigate known deadlock condition.
+	//        https://github.com/capnproto/go-capnproto2/issues/189
+	MaxConcurrentCalls: 64,
+	AnswerQueueSize:    64,
+}
+
+type ClusterServer struct {
+	node *cluster.Node
```

Comment:
We should define an package-scoped interface instead of passing the concrete `*cluster.Node`, as this greatly facilitates testing.

I'd suggest something like:

```
type ViewProvider interface {
    View() cluster.View
}
```